salesagility/SuiteCRM
# Resolve issue with email templates

## Description



Resolved issues not fixed in #5464 

## Motivation and Context

Prevented use of the email templates dropdown within the events module

## How To Test This

1. Head to email templates and make a new template, attempt to set the type to events
2. After hitting save, navigate to events module and attempt to create a new event.
3. See that on the email invites page that your template is not visible
4. Install fix and see again

## Types of changes


### Final checklist
